### PR TITLE
[breaking] added kubeVersion as a field in nova output and helm output

### PR DESCRIPTION
**Pull request by jhash1**
This PR fixes #

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### What's the goal of this PR?
Add functionality to expose the Kubernetes version field located in the chart.yaml file to provide visibility into K8s compatibility.
### What changes did you make?
I made changes to the output, clusterrepo and findscore packages. I added the kubeVersion field along with changes to the format table ouput

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -69,38 +69,36 @@ redis             15.4.1       15.5.5     true    false
...
-          "deprecated": false,
-          "helmVersion": "3",
-          "overridden": false
-        }
-    ]
-}
+  {
+    "release": "cert-manager2",
+    "chartName": "cert-manager",
+    "namespace": "cert-manager",
```
Comment (rbren) [inline docs/usage.md:84]: something changed here...

File: docs/usage.md [hunk 2]
```diff
@@ -69,38 +69,36 @@ redis             15.4.1       15.5.5     true    false
...
 
 ```
-Release Name      Chart Name        Namespace         HelmVersion    Installed    Latest     Old     Deprecated
-============      ==========        =========         ===========    =========    ======     ===     ==========
-goldilocks        goldilocks        goldilocks        3              3.3.1        4.0.1      true    false
-metrics-server    metrics-server    metrics-server    3              5.6.0        5.10.10    true    false
-redis             redis             redis             3              15.4.1       15.5.5     true    false
+Release Name      Chart Name        Namespace         HelmVersion   KubeVersion    Installed    Latest   Old  Deprecated
+============      ==========        =========         ===========   ===========    ===========  ======   ===  ==========
+cert-manager2     cert-manager      cert-manager          3         >= 1.22.0-0      v.1.12.0   v.1.12.2 true    false
```
Comment (rbren) [inline docs/usage.md:74]:
```suggestion
cert-manager     cert-manager      cert-manager          3         >= 1.22.0-0      v.1.12.0   v.1.12.2 true    false
```

**Comment by sudermanjr** [comment]
I marked this as `[breaking]` since it changes the output structure and format. This will require a major version change

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,26 +0,0 @@
-# Changelog
```
Comment (sudermanjr) [inline CHANGELOG.md:1]: Rather than removing this, I would recommend a deprecation notice at the top. See the reckoner repo for a similar situation.

**Pull request closed without merging**